andela/thorin-rc
# #158030499 Create landing page for shop

#### What does this PR do?

Creates the landing page for the Reaction Commerce Shop

#### Description of Task to be completed?

- Create component for flash deals
- Create component for different product categories
- Create footer component
- Add Categories to the Navbar
- Adds header for tags on the products grid page

#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `reaction` on the terminal. Visit the localhost address on port 3000. The landing page should be displayed .

#### Any background context you want to provide?

Add products to the database to see them appear on the landing page. Relevant tags for products are `e-books`, `fashion`, `tech`, `sports`, `games and toys`, `home`

#### What are the relevant pivotal tracker stories?

#158030499

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-06-21 at 3 13 18 pm" src="https://user-images.githubusercontent.com/25195857/41724740-1f2feede-7566-11e8-901a-18419e1a2e26.png">
<img width="1440" alt="screen shot 2018-06-21 at 3 39 43 am" src="https://user-images.githubusercontent.com/25195857/41695440-edc71b6a-7506-11e8-8c8e-180eb197d36a.png">
<img width="1440" alt="screen shot 2018-06-21 at 11 37 08 am" src="https://user-images.githubusercontent.com/25195857/41724852-66b82b7c-7566-11e8-9043-f955d81ca1bc.png">
